docs/guides/prisma.md

```diff
@@ -0,0 +1,360 @@
+
+## Adding Prisma to an existing project
+
+If you have a brownfield Platformatic DB project, you can introspect your database and generate the data model in your Prisma schema with the following command:
+
+```bash
+npx prisma db pull
+```
+
+The command will introspect your database and generate the [data model](https://www.prisma.io/docs/concepts/components/prisma-schema/data-model)
+
+Next, add the `@@ignore` attribute to the `versions` model to exclude it from the Prisma Client API:
+
+```diff
+model versions {
+  version BigInt    @id
+  name    String?
+  md5     String?
+  run_at  DateTime? @db.Timestamptz(6)
+
++  @@ignore
+}
+```
+
+To learn how you can evolve your database schema, you can jump to the [Evolving your database schema](#evolving-your-database-schema) section.
+
+## Adding Prisma to a new project
+
+Define a `Post` model with the following fields in your `schema.prisma` file:
+```groovy
+// prisma/schema.prisma
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgres"
+  url      = env("DATABASE_URL")
+}
+
+model Post {
+  id        Int      @id @default(autoincrement())
+  title     String
+  content   String?
+  published Boolean  @default(false)
+  viewCount Int      @default(0)
+  createdAt DateTime @default(now())
+}
+```
+
+The snippet above defines a `Post` model with the following fields and properties:
+- `id`: An auto-incrementing integer that will be the primary key for the model.
+- `title`: A non-nullable `String` field.
+- `content`: A nullable `String` field.
+- `published`: A `Boolean` field with a default value of false.
+- `viewCount`: An `Int` field with a default value of 0.
+- `createdAt`: A `DateTime` field with a timestamp of when the value is created as its default value.
+
+Next, run the following command to generate an up and down migration:
+
+```
+npx db-diff
+```
+
+The previous command will generate both an `up` and `down` migration based on your schema. The generated migration is stored in your `./migrations` directory. If you are currently using a different path to store the migration, you can provide the `--migrations-dir` flag followed by the path.
+
+You can then apply the generated migration using the Platformatic DB CLI:
+
+```bash
+npx platformatic db migrations apply
+```
+
+Platformatic uses [Postgrator](https://www.npmjs.com/package/postgrator) to run migrations. Postgrator creates a table in the database called `versions` to track the applied migrations. Since the `versions` table is not yet captured in the Prisma schema, run the following command to introspect the database and populate it with the missing model:
+
+```
+npx prisma db pull
+```
+
+Introspecting the database to populate the model prevents including the `versions` table in the generated down migrations.
+
+Your Prisma schema should now contain the `versions` model as follows:
```

Comment:
```suggestion
Your Prisma schema should now contain a `versions` model that is similar to this one (it will vary depending on the database system you're using):
```